continuousflex/protocols/protocol_genesis.py

```diff
@@ -313,6 +296,53 @@ def _defineParams(self, form):
                       label="projection angle image set  ", help='Image set containing projection alignement parameters',
                       condition="EMfitChoice==%i and projectAngleChoice==%i"%(EMFIT_IMAGES,PROJECTION_ANGLE_IMAGE))
 
+        form.addSection(label='MPI parallelization')
+
+        form.addParam('parallelType', params.EnumParam, label="How to process EM data ?", default=PARALLEL_MPI,
```

Comment:
Regarding this section, it seems that you are trying to locally reproduce the sections in host.conf in case parallel mpi is used. I am sure it works, but 1) it may be difficult to maintain 2) it might be tricky to combine with the queueing option offered in Scipion. Let's discuss that soon.